docs/xks/operator-guide/kubernetes/aks.md

```diff
@@ -62,6 +62,51 @@ az aks nodepool delete --cluster-name aks-dev-we-aks1 --resource-group rg-dev-we
 
 For additional information about updating the system nodes refer to [this blog post](https://pumpingco.de/blog/modify-aks-default-node-pool-in-terraform-without-redeploying-the-cluster/).
 
+## Update AKS
+
+### Useful commands in kubernetes
+
+When running patching AKS or just upgrading nodes it can be useful to watch your resources in kubernetes.
+
+```shell
+# Show node version
+kubectl get nodes -o jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.metadata.labels.kubernetes\.azure\.com\/node-image-version}{"\n"}{end}'
+
+# Watch nodes
+watch kubectl get nodes
+
+# Watch the status of all pods in the cluster
+kubectl get pods -A
```

Comment:
The explanation and the command are a bit at odds because this command is not actually "watching". I suspect adding `--watch` to the command makes sense here.